packages/@tinacms/graphql/src/schema/index.test.ts

```diff
@@ -9,10 +9,14 @@ describe('TinaSchema', () => {
     test('fetches correct collection with getCollectionByFullPath', async () => {
       const schema = createSchema()
 
-      const collection1 = schema.getCollectionByFullPath('content/test/foobar')
```

Comment:
@jeffsee55 Is this function ever called without the file extension?

Reply:
I think that's safe to assume.

> Disclaimer- I either didn't write this originally or remember almost nothing about it.